### ISSUE TITLE
Can't get hotkey to work

### ISSUE THREAD

**Issue by tuananh** [koekeishiya/kwm#512]
I'm using latest version (4.0.2) and default config in the example folder.

I've enabled Accessibility access for khd and kwm.

Take an example for this one

```
cmd - return             :    open -na /Applications/iTerm2.app
```

I tried `cmd+ return` or `ctrl+x cmd+return` but it doesnt work

**Comment by tuananh** [comment]
I forgot to say i'm using OS X 10.12

**Comment by dominiklohmann** [comment]
The syntax is `[mode +] mod1 [+ mod2 ...] - key : command` so `cmd - return` is what you were looking for.

You can watch the khd error logs with `tail -f /tmp/khd.err`, maybe there's something wrong?

**Comment by mackdata** [comment]
I'm a first time user and I have the same problem as @tuananh. I am glad to read your message, so I know I'm not crazy. Athough kwm does not respond to the hotkeys, it does respond to changing settings in the ``kwmrc`` file. I tried editing the padding; and I see that the padding changing. 

Some notes:
- I'm running osx 10.10.5
- Installed kwm with brew, and then move the ``kwmrc`` to ``~\.kwm\`` folder.

**Comment by dominiklohmann** [comment]
> Athough kwm does not respond to the hotkeys, it does respond to changing settings in the kwmrc file.

As of version 4.0.0, kwm no longer handles any hotkeys. This functionality has been moved to a [separate program called khd](https://github.com/koekeishiya/khd).

**Comment by jgillman** [comment]
@dominiklohmann Thank you so much! After upgrading nothing was working and I didn't know how to revert to an older version and was really disappointed. The README should be more clear about the dependency IMO. I would expect to see that in the Install section.

**Comment by dominiklohmann** [comment]
> The README should be more clear about the dependency IMO. I would expect to see that in the Install section.

Sounds like a good idea for a quick pull request if you want to improve the README file. 😉

**Comment by jgillman** [comment]
lol, I'm on it!

**Comment by jgillman** [comment]
Added a v1 PR with #518. Feedback on wording and/or if it's actually clear is welcome!

**Comment by jgillman** [comment]
For anyone else reading this and in the same situation as me, everything was working except for the prefix command which (after looking at the logs) reports `zsh:1: command not found: kwmc`

Looks like it's this issue: https://github.com/koekeishiya/khd/issues/9
And [this comment](https://github.com/koekeishiya/khd/issues/9#issuecomment-252153916) fixed it for me.

**Comment by koekeishiya** [comment]
> Thank you so much! After upgrading nothing was working and I didn't know how to revert to an older version and was really disappointed. The README should be more clear about the dependency IMO. I would expect to see that in the Install section.

To be fair, Kwm is released under the semver system, so when a change is performed to the major versioning tag,  I expect people to read the changelog that is posted along with the release.

As far as information regarding keybindings, this already exists [here](https://github.com/koekeishiya/kwm/tree/master/examples)

**Comment by jgillman** [comment]
@koekeishiya You're totally right, with a major version change breaking changes should be expected!

For me I updated via `brew` and didn't realize what I was getting into. If I was building from source I would have seen the changelog. The README in the example dir is really helpful but I'm personally not used to looking through the source (even if it's just an example dir). IDK where I fall on the power-user scale, but I suspect I'm not alone :)

Kwm is absolutely fantastic, btw. When I upgraded and it stopped working it was a blow because I love using it so much and really makes my day-to-day computer usage remarkably better!
